src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -646,6 +649,23 @@ CLParser::Usage(const std::string& msg)
                    "will include additional information.",
                    18)
             << std::endl;
+  std::cerr << FormatMessage(
+                   " --collect-metrics: Enables collection of server-side "
+                   "inference server metrics. Outputs metrics in the csv file "
+                   "generated with the -f option.",
```

Comment:
Can you mention that it is only included with the `--verbose-csv` and the default value is false?

Reply:
Added 👍